__tests__/app.test.js

```diff
@@ -231,4 +231,30 @@ describe('/api/articles', () => {
             });
         })
     })
+})
+
+describe('/api/comments/:comment_id', () => {
+    test('DELETE:204 deletes a comment and sends no body back', () => {
+        return request(app)
+        .delete('/api/comments/2')
+        .expect(204)
+    });
+
+    test('POST:400 sends an appropriate status and error message when given an invalid comment_id', () => {
+        return request(app)
+          .delete('/api/comments/banana')
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe('Bad request');
+          });
+    });
+
+    test('POST:404 sends an appropriate status and error message when given an non-existent comment_id', () => {
+        return request(app)
+          .delete('/api/comments/99988')
```

Comment:
Careful you don't get errors for out of index range 😄 I once saw someone test for 9999999999999999 and wondered why they didn't get a 404.